bot.py

```diff
@@ -601,19 +605,23 @@ async def on_message(self, message):
                             'timestamp': str(message.created_at),
                             'message_id': str(message.id),
                             'content': message.content,
-                            'type': 'thread_message', # TODO: Different message type in logviewer
+                            'type': 'mention',
                             'author': {
                                 'id': str(message.author.id),
                                 'name': message.author.name,
                                 'discriminator': message.author.discriminator,
                                 'avatar_url': str(message.author.avatar_url_as(static_format='png', size=1024)),
                                 'mod': False
                             },
-                            'attachments': attachments
+                            'attachments': attachments,
+                            'channel': {
```

Comment:
Might be a good idea to default `channel` to `None` if not a mention message or archive. Not sure ramifications for logviewer if it exists however